rundeckapp/grails-app/i18n/messages.properties

```diff
@@ -1915,7 +1915,7 @@ end=End
 job.editor.workflow.unsavedchanges.warning=Some changes to the workflow have not been completed.
 scheduledExecution.property.executionLifecyclePluginConfig.help.text=Selected Plugins will be enabled for this Job.
 scheduledExecution.executionLifecyclePlugins.pluginTypeNotFound.message=Execution Life Cycle plugin type "{0}" was not found or could not be loaded
-scheduledExecution.executionLifecyclePlugins.invalidPlugin.message=Invalid Configuration for Execution Life Cycle plugin: {0} - {1}
+scheduledExecution.executionLifecyclePlugins.invalidPlugin.message=Invalid Configuration for Execution Life Cycle plugin: {0}
```

Comment:
Included an improvement related to another change in this release. If this is a problem I can split the PR.